lib/mocktail/value/top_shelf.rb

```diff
@@ -1,65 +1,60 @@
-# The top shelf stores all cross-thread & thread-aware state, so anything that
-# goes here is on its own when it comes to ensuring thread safety.
```

Comment:
I removed this comment as the methods in the file now either:
* manipulate an instance that is only ever accessed by one thread
* explicitly synchronize on a Mutex